sympy/geometry/tests/test_ellipse.py

```diff
@@ -477,6 +477,10 @@ def test_section_modulus_and_polar_second_moment_of_area():
     assert e.section_modulus() == (18*pi, 6*pi)
     assert e.polar_second_moment_of_area() == 120*pi
 
+    e = Ellipse(Point(0,0), 2, 2)
```

Comment:
```suggestion
    e = Ellipse(Point(0, 0), 2, 2)
```
Similarly, add space after comma on line 482. See [PEP 8](https://www.python.org/dev/peps/pep-0008/).